ridgeback_msgs/msg/Status.msg

```diff
@@ -16,12 +16,14 @@ float32 mcu_temperature
 
 # Monitoring the run/stop loop. Changes in these values trigger an immediate
 # publish, outside the ordinarily-scheduled 1Hz updates.
-bool drivers_active
-bool driver_external_stop_present
-bool driver_external_stop_stopped
+bool stop_power_status  # True indicates everything is in order with the stop loop.
+bool stop_engaged  # True when a stop has been pressed on the robot.
+bool drivers_active  # True when a stop is asserted to the motor drivers.
```

Comment:
Isn't the logic the opposite for this? I still don't understand what the relationship is between these three. What does it means for the stop loop to be "in order"?